### PR TITLE
Added progressbar stories

### PR DESCRIPTION
**Pull request by Lisa159**

### CODE REVIEW

**Review by aweell — COMMENTED** [review]
It seems that the font is not loading correctly (Clearly evident in the helper-text example). I checked other component stories and it looks like (at least) is rendering a sans-serif. Is something we need to check?

**Comment by Lisa159** [comment]
> It seems that the font is not loading correctly (Clearly evident in the helper-text example). I checked other component stories and it looks like (at least) is rendering a sans-serif. Is something we need to check?

Looks like helperText has a wrong token in the code. It is using "fontFamily" token (which doesn't exist in this component) instead of "helperTextFontFamily" token. 
I will create an issue to fix this.

**Comment by raquelarrojo** [comment]
Tests without value will change all the times, so we could add value in all tests to avoid this.

**Comment by Lisa159** [comment]
> Tests without value will change all the times, so we could add value in all tests to avoid this.

Added values to examples.

File: lib/src/progress-bar/ProgressBar.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      <Title title="XxLarge margin" theme="light" level={4} />
+      <DxcProgressBar label="Margin xxlarge" margin="xxlarge" overlay={false} value={50} showValue/>
+    </ExampleContainer>
+  </>
+);
+
+export const ProgressBarOverlay = () => (
+  <ExampleContainer>
+    <Title title="Overlay" theme="light" level={4} />
+    <DxcProgressBar label="Overlay" helperText="Helper text" showValue value={50} />
```
Comment (raquelarrojo) [inline lib/src/progress-bar/ProgressBar.stories.jsx:56]: Here should be `theme="dark"`.

**Review by raquelarrojo — APPROVED** [review]

**Pull request merged**